src/test/ui/lint/invalid_value.stderr

```diff
@@ -212,11 +220,13 @@ LL |         let _val: Wrap<fn()> = mem::zeroed();
    |                                this code causes undefined behavior when executed
    |                                help: use `MaybeUninit<T>` instead, and only call `assume_init` after initialization is done
    |
-note: function pointers must be non-null (in this struct field)
+   = note: `Wrap<fn()>` must be non-null
```

Comment:
Previously I deliberately *first* checked the primitive type and then the rest... that's why the error changes here now.

"`Wrap<fn()>` must be non-null in this struct field" sound rather strange.